Sources/Common/Error.swift

```diff
@@ -181,18 +176,15 @@ extension OperaError: CustomStringConvertible {
         switch self {
         case .networking(let error, let request, _, _):
             var components = [String]()
-            if let error = error as? CustomStringConvertible {
-                components.append("Networking Error: \(error.description)")
-            }
-            if let error = error as? CustomDebugStringConvertible {
-                components.append(error.debugDescription)
-            }
+            components.append("Networking Error: \((error as CustomStringConvertible).description)")
+
```

Comment:
Yes I noticed it, but as this is the implementation of the `description` property I think it is ok. You can check that there are other minor differences with the implementation of `debugDescroption` like the JSON value which is not included in the error's `description`. What do you think?